### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by aflah02**
I have updated the README with the following - 
1. Added Links for Conda, Cython and AQR.
2. Added a brief description for what all the Dependencies do so people who are unfamiliar with them can atleast get a basic idea about them
3. Corrected PyPI spelling in one instance

### CODE REVIEW

**Comment by aflah02** [comment]
Hey
If any more changes are needed in the README just comment them below I'll do them as well.
Thanks :)

File: README.md [hunk 1]
```diff
@@ -100,15 +100,15 @@ pip install pandas
 ```
 
 ## Dependencies
-- [NumPy](https://www.numpy.org)
-- [python-dateutil](https://labix.org/python-dateutil)
-- [pytz](https://pythonhosted.org/pytz)
+- [NumPy - Adds support for large, multi-dimensional arrays, matrices and high-level mathematical functions to operate on these arrays](https://www.numpy.org)
+- [python-dateutil - Provides powerful extensions to the standard datetime module, available in Python 2.3+](https://labix.org/python-dateutil)
```
Comment (jreback) [inline README.md:104]: you can remove the , available from datetutil and pytz

**Review by jreback — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aflah02) [reply README.md:104]: Understood will do that, thanks for the suggestions @jreback

**Comment by aflah02** [comment]
I can't seem to understand why it failed. I think it is unrelated to my changes

**Comment by aflah02** [comment]
@jreback Just wanted to mention I've made the changes but some tests seem to fail however I think they are independent of my changes

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @aflah02 failures are unrelated to your PR

**Pull request merged**